vllm/model_executor/custom_op.py

```diff
@@ -56,7 +62,7 @@ def dispatch_forward(self):
         # NOTE(woosuk): Here we assume that vLLM was built for only one
         # specific backend. Currently, we do not support dynamic dispatching.
 
-        if envs.VLLM_TORCH_COMPILE_LEVEL >= CompilationLevel.INDUCTOR:
+        if not self.enabled():
```

Comment:
```suggestion
-        if not self.enabled():
+        enabled = self.enabled()
+        name = self.__class__.name
+        logger.debug("custom op %s %s", name,
+                     "enabled" if enabled else "disabled")
+        if not enabled:
```

add some logging here, so that we know which ops are used for the current model we run